css/style.css

```diff
@@ -0,0 +1,609 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+/* index.html - start*/
+
+.main-nav {
+    position: sticky;
+    top: 0.5rem;
+    z-index: 22222;
+    border: 3px solid #34dac9;
```

Comment:
Ca design, e puțin deranjant când meniul e deasupra altor elemente. Eu i-aș pune și un background.

Reply:
I-am pus un background-color ca sa se vada si efectul de hover si after.